content/docs/reference-events.md

```diff
@@ -52,70 +52,70 @@ function onClick(event) {
     console.log(eventType); // => "click"
   }, 0);
 
-  // Won't work. this.state.clickEvent will only contain null values.
+  // Não funciona. this.state.clickEvent irá conter apenas valores nulos.
   this.setState({clickEvent: event});
 
-  // You can still export event properties.
+  // Você ainda pode exportar as propriedades do evento.
   this.setState({eventType: event.type});
 }
 ```
 
-> Note:
+> Nota:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
-
-## Supported Events {#supported-events}
-
-React normalizes events so that they have consistent properties across different browsers.
-
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
-
-- [Clipboard Events](#clipboard-events)
-- [Composition Events](#composition-events)
-- [Keyboard Events](#keyboard-events)
-- [Focus Events](#focus-events)
-- [Form Events](#form-events)
-- [Mouse Events](#mouse-events)
-- [Pointer Events](#pointer-events)
-- [Selection Events](#selection-events)
-- [Touch Events](#touch-events)
-- [UI Events](#ui-events)
-- [Wheel Events](#wheel-events)
-- [Media Events](#media-events)
-- [Image Events](#image-events)
-- [Animation Events](#animation-events)
-- [Transition Events](#transition-events)
-- [Other Events](#other-events)
+> Se você deseja acessar as propriedades de um evento de forma assíncrona, você deve chamar o `event.persist()` no evento em questão, o que irá remover o evento sintético do acumulador e permitir referências ao evento a ser retido pelo código do usuário.
+
+## Eventos (Suporte) {#eventos-suporte}
+
+O React normaliza eventos para que se possa ter propriedades consistentes entre os navegadores.
+
+Os manipuladores de evento (_event handlers_) abaixo são acionados por um evento na fase de bolha (_bubbling_). Para registrar um manipulador de evento para a fase de captura, adicione `Capture` como sufixo do nome do evento. Por exemplo, ao invés de usar `onClick`, você usaria `onClickCapture` para manipular o evento de clique na fase de captura.
```

Comment:
```suggestion
Os manipuladores de evento (_event handlers_) abaixo são acionados por um evento na fase de propagação (_bubbling_). Para registrar um manipulador de evento para a fase de captura, adicione `Capture` como sufixo do nome do evento. Por exemplo, ao invés de usar `onClick`, você usaria `onClickCapture` para manipular o evento de clique na fase de captura.
```

I may be wrong, but I've never seen "fase de bolha" to refer tu bubbling events. Maybe as the other kind of propagation is "capturing" which is pretty straightforward to translate, we should keep just "propagação" with bubbling in parenthesis, what do you think?

Reply:
much better. I'm making a collection of words and its translations and will add that as well. thanks for reviewing